src/Dnn.Modules.Vendors/EditAffiliate.ascx.designer.cs

```diff
@@ -28,7 +28,7 @@ public partial class EditAffiliate {
         /// Auto-generated field.
         /// To modify move field declaration from designer file to code-behind file.
         /// </remarks>
-        protected global::DotNetNuke.Web.UI.WebControls.DnnDatePicker StartDatePicker;
+        protected global::System.Web.UI.WebControls.TextBox StartDatePicker;
```

Comment:
If this was changed to a textbox only, it will make it very hard to enter dates in a proper format as it varies by culture, people will also be writing dates in different formats. Is there some javascript to make it a picker that is missing in the PR or changes on the .ascx file missing?

Reply:
One solution I can think of would be using an HTML5 <input type="date" /> maybe.

A quick google search gives me a few examples such as this one https://www.c-sharpcorner.com/UploadFile/4b0136/introducing-html-5-date-input-type-in-Asp-Net/

Reply:
The EditAffiliates.ascx and EditBanner.ascx forms have been updated based on the example you provided. I'm not sure why jQuery UI datepicker wouldn't work as a solution. I also would have thought that DNN would have a datepicker that could be referenced in Module development but (again) haven't found an example.
Any tips you have on installing cake to be able to build this in Release would be awesome.